bms-monitor.js

```diff
@@ -1,18 +1,44 @@
 const {expect} = require('chai');
 
-function batteryIsOk(temperature, soc, charge_rate) {
+function checkBatteryTemperature(temperature) {
+    console.log("checkBatteryTemperature: ",temperature)
     if (temperature < 0 || temperature > 45) {
-        console.log('Temperature is out of range!');
+        console.log('Temperature is out of range!', temperature);
         return false;
-    } else if (soc < 20 || soc > 80) {
+    }
+    
+        return true;
+    
+}
+function checkBatterySOC(soc) {
+    console.log("checkBatterySOC: ",soc)
+    if (soc < 20 || soc > 80) {

```

Comment:
usage of hard coded values will lead to issues during maintenance. if it gets used in many places and then a change in one place needs to be consistent with all others. There is a scope for improvement here.